src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/Internal/Topics.cs

```diff
@@ -329,8 +332,15 @@ protected override bool Business(object message)
     /// </summary>
     internal static class Utils
     {
+        private record MakeKeyInfo(ActorPath Path, string Topic);
```

Comment:
Stupid Question, have we checked this vs a `private record readonly struct` as far as allocations vs performance?

Reply:
Not yet, I'll give that a go :+1:

Reply:
Looks like converting this to `private readonly record struct` fixes the `MakeKey()` SOH memory allocation problem :+1: